sdk/eventhubs/azure-messaging-eventhubs/CMakeLists.txt

```diff
@@ -84,16 +84,21 @@ set(
     src/consumer_client.cpp
     src/event_data.cpp 
     src/event_data_batch.cpp
+    src/partition_client_models.cpp
     src/private/package_version.hpp
+    src/private/eventhub_constants.hpp
+    src/private/eventhub_utilities.hpp
```

Comment:
unrelated nit: but should we be consistent in file names and use plural
`eventhubs_constants.hpp`